### DISCUSSION TITLE
failed to convert file: failed to find schema for float64

### DISCUSSION THREAD

**Discussion by furynerd** [foxglove/mcap#1279] [General] [answered]
i successfully converted ros2 db3 bag to mcap, but with some bag i recently received, i face error 
```
failed to convert file: failed to find schema for float64
```

as the bag recording and msg define is out of my access, is there any hint about the above error?
thanks

**Comment by jtbandes** [comment]
Could you share a bag file that we can use to reproduce this error?

**Reply by furynerd** [reply, marked as answer]
sorry i'm not allowed to share the bags out...but i checked topics one by one and located the dubious topic (luckly not my interested topic ), and wrote a script to strip out that topic, then i could carried on my works. thanks again for response
